router/http_test.go

```diff
@@ -401,6 +409,65 @@ func (s *S) TestStickyHTTPRoute(c *C) {
 	}
 }
 
+func (s *S) TestStickyHTTPRouteWebsocket(c *C) {
+	wshandler := func(w http.ResponseWriter, req *http.Request) {
+		w.Write([]byte("ok " + strings.TrimPrefix(req.URL.Path, "/")))
```

Comment:
because the request path is used for the response body, it's possible for a request to `srv2` with path `/1` to respond with `"ok 1"`.